src/components/implementation/no_interface/scheddev/sl.c

```diff
@@ -104,11 +104,10 @@ sl_thd_wakeup(thdid_t tid)
 }
 
 void
-sl_thd_yield(thdid_t tid)
+sl_thd_yield_cs_exit(thdid_t tid)
```

Comment:
I need this functionality to avoid a race in sl_lock_lock, where a timeslice could sometimes be wasted.